### PR TITLE
add metric of running compaction per database

### PR DESCRIPTION
**Pull request by pesimon**

### CODE REVIEW

**Comment by gesellix** [comment]
Note to self: [link to relevant api docs](http://docs.couchdb.org/en/2.2.0/api/database/common.html#get--db)

File: lib/couchdb-stats.go [hunk 1]
```diff
@@ -103,6 +103,8 @@ type DatabaseStats struct {
 	DataSize         float64 `json:"data_size"`
 	DocCount         float64 `json:"doc_count"`
 	DocDelCount      float64 `json:"doc_del_count"`
+	CompactRunningImport   bool    `json:"compact_running"`
```
Comment (gesellix) [inline lib/couchdb-stats.go]: Can you please rename the field to `CompactRunningBool`?

**Comment by pesimon** [comment]
@gesellix pls tak a look and merge

**Comment by gesellix** [comment]
thanks!

**Comment by gesellix** [comment]
the changes are available with `gesellix/couchdb-prometheus-exporter:18`.

**Pull request merged**